service/src/test/java/uk/gov/hmcts/reform/fpl/service/email/content/cmo/CMOReadyToSealContentProviderTest.java

```diff
@@ -0,0 +1,78 @@
+package uk.gov.hmcts.reform.fpl.service.email.content.cmo;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ContextConfiguration;
+import uk.gov.hmcts.reform.fpl.enums.JudgeOrMagistrateTitle;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.HearingBooking;
+import uk.gov.hmcts.reform.fpl.model.Respondent;
+import uk.gov.hmcts.reform.fpl.model.RespondentParty;
+import uk.gov.hmcts.reform.fpl.model.common.JudgeAndLegalAdvisor;
+import uk.gov.hmcts.reform.fpl.model.notify.cmo.CMOReadyToSealTemplate;
+import uk.gov.hmcts.reform.fpl.service.email.content.AbstractEmailContentProviderTest;
+import uk.gov.hmcts.reform.fpl.utils.EmailNotificationHelper;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.fpl.enums.HearingType.CASE_MANAGEMENT;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.wrapElements;
+
+@ContextConfiguration(classes = {CMOReadyToSealContentProvider.class})
+class CMOReadyToSealContentProviderTest extends AbstractEmailContentProviderTest {
+    @Autowired
+    private CMOReadyToSealContentProvider contentProvider;
+
+    @MockBean
+    private EmailNotificationHelper notificationHelper;
```

Comment:
Really same reason as above - I wouldn't mock it. With mocking you actually have more setup to do. If you add EmailNotificationHelper.class, TimeConfiguration.class to the ContextConfiguration, this test passes without this line and (lol @ BeforeEach tags someone...) 

Again I would like you to develop your own opinion, but I think mocking "helper" classes is not necessary. They are increasing the complexity (marginally), of test configuration, but don't increase the test maintainability.

Reply:
I was trying to lessen the dependance on starting up spring here. Ideally `EmailNotificationHelper` wouldn't be a bean and #1433 is looking to resolve that. With this story I was trying out the idea that unit tests, unless really necessary, shouldn't start up spring as that could slow things down quite a bit relatively speaking.